### ISSUE TITLE
Specify Card Logos

### ISSUE THREAD

**Issue by crookedneighbor** [braintree/braintree-web-drop-in#688]
> If I can ask one question: I was searching in the Drop-in UI Developer guides; however, I can't find a way how to remove some of the logos of cards visible on the right-side of the Pay with card form (see highlight in the image below). I'm asking this because we accept only Visa and Mastercard (and no Amex or others), and by having all logos there creates confusion and generates unsuccessful payments. Many thanks.
>
> ![Braintree - cards](https://user-images.githubusercontent.com/473007/104808773-bc699f80-57e8-11eb-8bcd-e6350a47c528.png)

_Originally posted by @studios in https://github.com/braintree/braintree-web-drop-in/issues/613#issuecomment-761536739_

**Comment by crookedneighbor** [comment]
These logos _should_ be determined by the configuration on your merchant account. I _think_ it may be pulled only from the configuration from your _default_ merchant account, so if you're specifying a merchant account in your client token generation, and that merchant account does not support Discover, that could be the reason you're seeing this.

**Comment by crookedneighbor** [comment]
If that's the case, we'll have to provide someway to override the logos provided. 

@studios, If you need a temporary workaround, you can target:

```css
[data-braintree-id="discover-card-icon"] {
  display: none !important;
}

**Comment by oscarleonnogales** [comment]
We've released a fix for this as of version 1.38.0

You can selectively disable different card brands by setting the appropriate fields in the `card.overrides.fields.number.supportedCardBrands` object when creating the Braintree dropin instance, disabling the card brands this way will now also prevent the respective icon from rendering.

Example shown below.

`braintree.dropin.create({
        authorization: 'your_token',
        container: '#dropin-container',
        card: {
          overrides: {
            fields: {
              number: {
                supportedCardBrands: {
                  visa: true,
                  mastercard: true,
                  "american-express": false
                },
              }
            }
          }
        }
      })`
